src/Sarif.Viewer.VisualStudio/Tags/TextViewCaretListener.cs

```diff
@@ -64,13 +64,7 @@ private void UpdateAtCaretPosition(CaretPosition caretPosition)
         {
             SnapshotPoint caretSnapshotPoint = caretPosition.BufferPosition;
 
-            // If the caret is at the end of the file, then the caret cannot be inside a tag.
-            if (caretSnapshotPoint.Position + 1 >= this.textView.TextBuffer.CurrentSnapshot.Length)
-            {
-                return;
-            }
-
-            NormalizedSnapshotSpanCollection normalizedSnapshotSpanCollection = new NormalizedSnapshotSpanCollection(new SnapshotSpan(caretSnapshotPoint, 1));
+            NormalizedSnapshotSpanCollection normalizedSnapshotSpanCollection = new NormalizedSnapshotSpanCollection(new SnapshotSpan(caretSnapshotPoint, caretSnapshotPoint));
```

Comment:
>caretSnapshotPoint [](start = 154, length = 18)

Ah, without the ", 1"

Reply:
And since it's not clear from the overloaded constructor... This is a start, end not a length.

So I'll used named parameters.

---
In reply to: [503598671](https://github.com/microsoft/sarif-visualstudio-extension/pull/219#discussion_r503598671) [](ancestors = 503598671)